### DISCUSSION TITLE
HCF Teamfights

### DISCUSSION THREAD

**Discussion by Nqvn** [diamond-rip/Eden#471] [Ideas]
*Before you read this, if hcf teamfights is already possible with different kits (Bard, archer etc) then please let me know.*

Hello, I'm running a hcf server with practice, and I wanted to add a hcf teamfight kit for tournaments and such, however I couldn't find or figure out how to setup a kit that has different loadouts for the same kit, and how to select them (So selecting Archer, Bard, Diamond kits before match) If this is not added right now then please do, because different loadouts in kits would also make kits like Hcf Trapping possible. Again if this is already possible then please let me know.

**Comment by Nqvn** [comment]
Atm I'm gonna go with adding Bard and Archer as extra items in kit editor however that's not the most convenient thing in the world

**Reply by GoodestEnglish** [reply]
Right now it is not possible to create kits with multiple selection.
